lib/src/voip/voip.dart

```diff
@@ -479,6 +485,12 @@ class VoIP {
     // by terminate.
     currentCID = VoipId(roomId: room.id, callId: callId);
 
+    if (confId == null) {
+      await delegate.registerListeners(newCall);
```

Comment:
What happens if you always register the listeners at this point? Could you potentially remove the second registerListeners call below?

Reply:
Well, the other reference is used in a different function `inviteToCall` which gets called for incoming calls. And this usage is for `onCallInvite`.

Reply:
ah github diff :D